website/docs/plugins/nevergrad_sweeper.md

```diff
@@ -52,23 +54,24 @@ hydra:
           init: 0.02
           step: 2.0
           log: true
-        # a custom nevergrad parameter evaluated at runtime (avoid using this whenever possible)
-        dropout: Scalar(lower=0.0, upper=1.0)
         # an integer scalar going from 4 to 16
-        # init is set to the middle of the range
-        # step is set to a sixth of the range
+        # init and step parameters could also be provided,
+        # by default init is set to the middle of the range
+        # and step is set to a sixth of the range
         batch_size:
           lower: 4
           upper: 16
           integer: true
+        # a custom nevergrad parameter evaluated at runtime (avoid using this whenever possible)
+        dropout: Scalar(lower=0.0, upper=1.0)
 
 db: cifar
 lr: 0.01
-dropout: 0.6
 batch_size: 8
+dropout: 0.6
 ```
 
-The application `main` returns a float which we want to minimimize, the minimum is 0 and reached for:
+The the function decorated with `@hydra.main()` returns a float which we want to minimize, the minimum is 0 and reached for:
```

Comment:
Should mention how to maximize somewhere in the readme.
Maybe as a commented out line in the config node above.

Reply:
added to the conf